addons/mail/static/src/messaging/component/chat_window/chat_window.js

```diff
@@ -130,9 +136,8 @@ class ChatWindow extends Component {
      * @private
      */
     _update() {
-        if (this.chatWindow.doIsFocus) {
+        if (this.chatWindow && this.chatWindow.isDoFocus) {
```

Comment:
Yes, this was the main fix (the typo) of my branch

Reply:
in EntityChatWindow we use _cycleNextVisibleUnfoldedChatWindow which do not return anything, and use its return to focus.... It's weird, I modified that in my PR too, also, I wrote a test